plugins/Zeroname/updater/zeroname_updater.py

```diff
@@ -69,7 +68,7 @@ def processNameOp(domain, value, test=False):
     # Note: Requires the file data/names.json to exist and contain "{}" to work
     names_raw = open(names_path, "rb").read()
     names = json.loads(names_raw)
-    for subdomain, address in data["zeronet"].items():
+    for subdomain, address in list(data["zeronet"].items()):
```

Comment:
`list` is not required around `items()` in a `for` loop

Reply:
I think is better to leave it this way.

Reply:
I agree with @imachug.  The `list()` increases memory use and slows processing time which is why Python 3 moves towards iteration over fixed lists.  `list()` would be required if the contents of `data["zeronet"]` was being changed inside the for loop.